_component_design/radio-buttons.md

```diff
@@ -7,157 +7,140 @@ usage: >
-  <div class="ui error message">
-    There is an error with the response to this question.
-  </div>
+
 </div>
+
+## Specifications
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Composition
+
+A radio button group consists of a Section Label, followed by pairs of radio buttons and radio button labels.
+
+#### Labeling:
+
+- Labels should start with a capital letter.
+- Labels should not have punctuation at the end, such as a period, unless more than one sentence is used. 
+
 {% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Radio button group composition"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/radio-buttons/radio-composition.svg)
+{% endfigure %}
+{% endcolumn %}
+
 </div>
 
-# Radio Button Group
+### Spacing
 
-Use when selecting a **single item** from a set if it's important for the user
-to see all options at once. This alternate can be used instead of regular radio
-buttons if the options are **simple**, limited in number and individual length.
+<div class="hxRow">
 
-## Enabled
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+A title should always be located above the radio buttons. More context can be provided under the title (subtitle) but it is optional (shown here as “Select an option” on left). If subtitle is not needed, use specs in the design on the right.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Radio button spacing"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/radio-buttons/radio-spacing.svg)
+{% endfigure %}
+{% endcolumn %}
 
-<div class="ui form">
-  <div class="field">
-    <label>Select potential maintenance days</label>
-    <div class="ui buttons single">
-      <button class="ui button ds-btn-med-sec active">Mon</button><button class="ui button ds-btn-med-sec">Tues</button><button class="ui button ds-btn-med-sec">Wed</button><button class="ui button ds-btn-med-sec">Thurs</button><button class="ui button ds-btn-med-sec">Fri</button><button class="ui button ds-btn-med-sec">Sat</button><button class="ui button ds-btn-med-sec">Sun</button>
-    </div>
-    <div class="ui message subtext">
-      Here is some additional help text to explain these choices.
-    </div>
-  </div>
 </div>
 
-## Disabled
+### States
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Overview
+
+A title should always be located above the radio buttons. More context can be provided under the title (subtitle) but it is optional (shown here as “Select an option” on left). If subtitle is not needed, use specs in the design on the right.
+
+Best practice form design would not allow a radio button to ever exist in error state, so try to avoid it at all cost.
```

Comment:
Form design best practices prohibit a radio button from existing in an error state. Avoid this situation.

Reply:
updated.